applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -71,9 +71,19 @@ def GetDefaultParameters(cls):
             },
             "bfecc_convection" : false,
             "bfecc_number_substeps" : 10,
```

Comment:
These should be into the `levelset_convection_settings`. As the settings would differ between the BFECC and convection-diffusion transport solver, we need to have two sets of defaults. I suggest proceeding as we did in the embedded solver with the distance modification.

Reply:
After a brief discussion @mrhashemi and I agreed to do this in a future PR so to make sure that we keep retrocompatibility. This would ease backtracking eventual errors.

Reply:
We can do this in a future PR where we address the semi-Lagrangian approach implemented in bfecc_convection.h.